src/web/mjs/connectors/Cycomi.mjs

```diff
@@ -8,54 +8,133 @@ export default class CyComi extends Connector {
         super.id = 'cycomi';
         super.label = 'CyComi';
         this.tags = ['manga', 'japanese'];
-        this.url = 'https://cycomi.com';
+        this.url = 'https://web.cycomi.com';
     }
 
     async _getMangaFromURI(uri) {
         const request = new Request(uri, this.requestOptions);
-        const id = uri.pathname;
-        const title = (await this.fetchDOM(request, 'div.title-texts > h3'))[0].textContent.trim();
+        const path = uri.pathname.split("/");
+        const id = path[path.length-1];
+        const title = (await this.fetchDOM(request, 'h2.css-m40re5'))[0].textContent.trim();
         return new Manga(this, id, title);
     }
 
     async _getMangas() {
+        const uri = new URL('/api/title/serialization/list', this.url);
+        const request = new Request(uri, this.requestOptions);
+        const response = await this.fetchJSON(request, 'div.card-content > a.card');
         const mangaList = [];
-        for(let page = 0; page < 8; page++) {
-            const mangas = await this._getMangasFromPage(page);
+        for(let page = 0; page < 8; page += 1) {
+            const mangas = response.data.serialization[""+page].titles.map(title => {
+                return {
+                    id: title.titleId,
+                    title: title.titleName
+                };
+            });
             mangaList.push(...mangas);
         }
         return mangaList;
     }
 
-    async _getMangasFromPage(page) {
-        const uri = new URL('/fw/cycomibrowser/title/serialization/' + page, this.url);
-        const request = new Request(uri, this.requestOptions);
-        const data = await this.fetchDOM(request, 'div.card-content > a.card');
-        return data.map(element => {
-            return {
-                id: this.getRootRelativeOrAbsoluteLink(element, this.url),
-                title: element.querySelector('.card-texts-title').textContent.trim()
-            };
-        });
+    async _getChapters(manga) {
+        let cursor = undefined;
+        const chapterList = [];
+        do {
+            const [chapters, nextCursor] = await this._getChaptersFromCursor(manga, cursor);
+            chapterList.push(...chapters);
+            cursor = nextCursor;
+        } while (cursor);
+        return chapterList;
     }
 
-    async _getChapters(manga) {
-        const uri = new URL(manga.id, this.url);
+    async _getChaptersFromCursor(manga, cursor) {
+        const params = new URLSearchParams({
+            limit: 20,
```

Comment:
It that possible to increase that number?  So we will perform less requests on their server.